libindy/src/services/pool/types.rs

```diff
@@ -465,6 +465,30 @@ impl PoolConfig {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct PoolOpenConfig {
```

Comment:
@vimmerru @dsurnin @ashcherbakov do you have more suggestions about items to configure?